.github/workflows/continuous-integration.yml

```diff
@@ -86,6 +86,30 @@ jobs:
            -DCMAKE_PREFIX_PATH:PATH="$PWD/../../AV_install;${DEPS_INSTALL_DIR}"
           make -j$(nproc)
 
+      - name: Functional Tests with Meshroom
+        working-directory: ./functional_tests
+        run: |
+          # clone required repos
+          git clone --branch develop https://github.com/alicevision/Meshroom.git
+          git clone --branch main https://github.com/alicevision/dataset_panoramaFisheyeHdr.git
+          cd Meshroom/
+          # select meshroom branch
+          meshroom_avBranch=$(git ls-remote --heads https://github.com/alicevision/Meshroom.git $GITHUB_HEAD_REF | cut -f 1)
+          if [ $meshroom_avBranch != "" ]; then git checkout $meshroom_avBranch; echo "Use Meshroom/$GITHUB_HEAD_REF"; fi
+          export MESHROOM_INSTALL_DIR=$PWD
+          export PYTHONPATH=$PWD:${PYTHONPATH}
+          export ALICEVISION_ROOT=$PWD/../../../AV_install/
+          export PATH=$PATH:$PWD/../../../AV_install/bin:$PWD/../../../AV_install/lib
+          export ALICEVISION_SENSOR_DB=$PWD/../../../AV_install/share/aliceVision/cameraSensors.db
+          export ALICEVISION_LENS_PROFILE_INFO=""
+          export LD_LIBRARY_PATH=$PWD/../../../AV_install/lib64:${DEPS_INSTALL_DIR}/lib64:${DEPS_INSTALL_DIR}/lib:${LD_LIBRARY_PATH}
```

Comment:
Maybe better to set globally?

Reply:
CI error if PATH or LD_LIBRARY_PATH set globally but ok for ALICEVISION_XXXX variables.